File: presentation/lite/src/main/java/com/github/anrimian/musicplayer/lite/LiteApp.java
package com.github.anrimian.musicplayer.lite;

import com.github.anrimian.musicplayer.App;
import com.github.anrimian.musicplayer.lite.di.LiteComponents;

public class LiteApp extends App {

    @Override
    protected void initComponents() {
        LiteComponents.init(getApplicationContext());
    }

}

File: app/lite/src/main/java/com/github/anrimian/musicplayer/lite/LiteApp.kt
package com.github.anrimian.musicplayer.lite

import android.content.Context
import com.github.anrimian.musicplayer.App
import com.github.anrimian.musicplayer.lite.di.LiteComponents

class LiteApp : App() {

    override fun initComponents(appContext: Context) {
        LiteComponents.init(appContext)
    }

}
